### ISSUE TITLE
France deaths seems not accurate

### ISSUE THREAD

**Issue by mathiastiberghien** [CSSEGISandData/COVID-19#1047]
Hi, thanks for the great works.
I tried to use the time series to create charts, and I discovered that at least for France (state), France (country) the data is not accurate. It says that the number of death is stable for 3 days (148),
but when I check [here](https://coronavirus.politologue.com/), I can see that the number of deaths is 264

**Comment by JiPiBi** [comment]
if your search in the issues with keyword France , you'll find many threads on similar remarks (for countries like Italy quite the same)

But it seems that this site doesn't changes these anomalies , wait and see

**Comment by jeanthib** [comment]
Ok but parhaps we can help by daily updating this site. Every days the true numbers are published around 7:30 pm

**Comment by mathiastiberghien** [comment]
Looking in the issues as you suggested, I found that links that seems better for french data: https://raw.githubusercontent.com/opencovid19-fr/data/master/dist/chiffres-cles.json

**Comment by JiPiBi** [comment]
@mathiastiberghien  I agree , the sources in France are the same that I have yet : Sante Publique and regional ARS . 

@jeanthib  as I said  values in the time series (and daily report too) are rarely updated even obviuously wrong , look for confirmed at 12th for France and Italy  , look at lasts days for deaths in France 
My only goal when I put official data here for France ,  is to give information for people that have implemented fixes for an update of their corrective dictionaries

**Comment by flccrakers** [comment]
> Looking in the issues as you suggested, I found #905 which has seems to have a better link for french data: https://raw.githubusercontent.com/opencovid19-fr/data/master/dist/chiffres-cles.json

I guess at this stage the exact number are not so important, no ? The main tendency is what is needed even if it would be good to have the most correct number. 
I also try to make chart by countries, you can go there: http://covid-by-contries.alwaysdata.net/ the idea is to try to watch the evolution of the curve, see when it start to flatten ;-).
PS: I know that my comment has nothing to do here but didn't find a way to contact you directly.

**Comment by mathiastiberghien** [comment]
If you look Italian decease curve and French decease curve. You can see that the curve are parallel with 10 days of difference. I read an article about it few days ago.

I just wanted to check if the curve are still parallel or not: What I did: take Italian data and French data and add a 10 days offset to the French data.

 

Unfortunately, it seems that they still are.

 

Here’s a link to see the graph: Vega covid Italy vs France (10 days offset for France)

 



 

In blue you have the Italy

In red the France

 

With the wrong data it seems that the French deceased is stable but with the real data you can understand what we should expect for the next week => that’s why it was important to me

 

Last day Italy had 475 deceases in one day.

In France the average daily decease rate (all deceases) for 2019 was around 1700)

 

I was very dubitative when they start to talk about that virus, but seeing that charts and number helped me understand the gravity of the situation.

**Comment by sfcfs** [comment]
@flccrakers : The per country evolution view is a great idea. Was missing that feature on the JHU site. Wondering if you could order the list of countries alphabetically, right now it is not and makes it difficult to find different countries. Also, wondering if you can provide an aggregate for USA, it seems that you have all the states but not a country level view.
Thanks for all your help!

**Comment by JiPiBi** [comment]
@flccrakers  the tendancy between 148  deaths in JHU for the 18th and  175  for the same day is not exactly the same ..... some days  the differences are quite huge , so on my side fixes are necessary

**Comment by flccrakers** [comment]
> @flccrakers : The per country evolution view is a great idea. Was missing that feature on the JHU site. Wondering if you could order the list of countries alphabetically, right now it is not and makes it difficult to find different countries. Also, wondering if you can provide an aggregate for USA, it seems that you have all the states but not a country level view.
> Thanks for all your help!

@sfcfs 
Thanks for your suggestions.
Yes , I will order alphabetically and also propose a list aggregated. I only started this project yesterday night so, there is still a lot of things to add.

**Comment by flccrakers** [comment]
@JiPiBi yes agree with you, it should be corrected.

**Comment by flccrakers** [comment]
@mathiastiberghien I didn't  find your link in your message, if you can share it. But let's do that outside of this issue thread -> you can contact me frank.hoonakker@gmail.com

**Comment by JiPiBi** [comment]
I finally found a source https://twitter.com/BFMTV/status/1240342002052538374 in which the French Health Director clearly said 264 , but on the site it was written 244 ...

**Comment by sfcfs** [comment]
#flccrakers - love the updates to the site! Well done! Really useful. The info at city level for USA must have had an issue, because it shows 0 cases in many cities, like San Francisco, where the real number is 79. You may want to remove that level of granularity till it the data is accurate.

**Comment by flccrakers** [comment]
> #flccrakers - love the updates to the site! Well done! Really useful. The info at city level for USA must have had an issue, because it shows 0 cases in many cities, like San Francisco, where the real number is 79. You may want to remove that level of granularity till it the data is accurate.

Thank you !
Yes, I'm thinking about cleaning the data, like you say, remove point in certain condition, for example, if you have a point to 0 after a point not to 0, or having a point to between two non 0 point. I will update tomorrow I think. 

Best

**Comment by jeanthib** [comment]
Or if a figure is the same than the previous one.Jean Thibaud06 11 71 71 55https://www.smartitude.frLe 21 mars 2020 20:27, Frank <notifications@github.com> a écrit :



#flccrakers - love the updates to the site! Well done! Really useful. The info at city level for USA must have had an issue, because it shows 0 cases in many cities, like San Francisco, where the real number is 79. You may want to remove that level of granularity till it the data is accurate.



Thank you !

Yes, I'm thinking about cleaning the data, like you say, remove point in certain condition, for example, if you have a point to 0 after a point not to 0, or having a point to between two non 0 point. I will update tomorrow I think.

Best

**Comment by flccrakers** [comment]
> Or if a figure is the same than the previous one.Jean Thibaud06 11 71 71 55https://www.smartitude.frLe 21 mars 2020 20:27, Frank <notifications@github.com> a écrit : #flccrakers - love the updates to the site! Well done! Really useful. The info at city level for USA must have had an issue, because it shows 0 cases in many cities, like San Francisco, where the real number is 79. You may want to remove that level of granularity till it the data is accurate. Thank you ! Yes, I'm thinking about cleaning the data, like you say, remove point in certain condition, for example, if you have a point to 0 after a point not to 0, or having a point to between two non 0 point. I will update tomorrow I think. Best —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

Done. Thanks for the ideas.
I'm know trying fixing data for France, by getting more reliable (and more frequently updated) data sources.

**Comment by flccrakers** [comment]
> > Or if a figure is the same than the previous one.Jean Thibaud06 11 71 71 55[https://www.smartitude.frLe 21](https://www.smartitude.frLe%C2%A021) mars 2020 20:27, Frank [notifications@github.com](mailto:notifications@github.com) a écrit : #flccrakers - love the updates to the site! Well done! Really useful. The info at city level for USA must have had an issue, because it shows 0 cases in many cities, like San Francisco, where the real number is 79. You may want to remove that level of granularity till it the data is accurate. Thank you ! Yes, I'm thinking about cleaning the data, like you say, remove point in certain condition, for example, if you have a point to 0 after a point not to 0, or having a point to between two non 0 point. I will update tomorrow I think. Best —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
> 
> Done. Thanks for the ideas.
> I'm know trying fixing data for France, by getting more reliable (and more frequently updated) data sources.

Done, I've applied a fix on french data to get update faster with more accuracy (every day at 14:00, from that repo: [https://github.com/opencovid19-fr/data](https://github.com/opencovid19-fr/data) who get sources from french gouvernement).

**Comment by JiPiBi** [comment]
@flccrakers 
I dont know if you have completely looked at the data in this french site , if not you can get interesting data recorded from daily ARS reports  . There is a dedicated folder in yaml format by region
